pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -266,12 +266,6 @@ type TaskRunStatusFields struct {
 	// +listType=atomic
 	RetriesStatus []TaskRunStatus `json:"retriesStatus,omitempty"`
 
-	// Results from Resources built during the TaskRun. currently includes
-	// the digest of build container images
-	// +optional
-	// +listType=atomic
-	ResourcesResult []PipelineResourceResult `json:"resourcesResult,omitempty"`
```

Comment:
I'd retain this field for backwards compatibility. We never marked it as deprecated, and similar to the TaskRun status we should avoid removing this until we're sure most clients will no longer need this for older objects / server versions.

Reply:
Thanks @wlynch for the guidance from the users' perspective.
totally understand the concerns for backwards compatibility concerns for older versions. It was debating for this field as it is related with PipelineResources that has long deprecated and just been removed.

For moving forward, would changing from **removal** to **tomb-stoning** the `taskrun.status.resourcesresult` field make sense in this PR as the followup of migrating off the usage of the `PipelineResourceResult` struct etc as the plan for `PipelineResourceResult` migration?

Reply:
And shall we announce deprecation/ tomestoning this field since the resourcesResult is not used any where in the codebase, and it won't interfere with the usage:)

Reply:
SGTM! We should definitely tombstone + mark as deprecated :)